### ISSUE TITLE
Venue allocation view should indicate conflicts

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat#770]
The venue allocation view does not indicate conflicts, contrary to adjudicator allocation. To minimize the need to go to the draw list (where conflicts are listed) to see whether constraints were matched.

**Comment by philipbelesky** [comment]
Yup, this is something we had initially planned but never really found the time for. This is mostly because, from my perspective, because this functionality is rarely used — at large tournaments room allocations are basically automatic (because they use constraints) so there is little need to manually swap and thus little risk of creating or overwriting a constraint.

If you do want to give it a shot the existing Vue components for highlighting information on rollover may not be abstracted enough to handle this use case. However they will also probably be reworked in the mid/near future as part of a new allocations page.

**Comment by tienne-B** [comment]
Being bold and adding it to the Adj Allocation project, as these highlights are now advertised on the allocation page but not implemented. My Vue skills are not advanced enough for this; tried starting it to no avail.

**Comment by tienne-B** [comment]
Looking back at this task, as I was just working on venue highlights. A way I'd approach the task is to add venue_constraints to adjudicator / team Vue objects, then check that if their arrays are not empty, the venue they are assigned has a listed (for both the participant and venue) category.

This would require an `InlineAdjudicator` object though to replace the straight text currently used for the adjs.

**Comment by philipbelesky** [comment]
Adding an `InlineAdjudicator` makes sense. I was hoping that these could be relatively cleanly modelled of how conflicts are modelled and tested, but recognise that there are a few differences that might make it tricky.
